release_gap.sh

```diff
@@ -9,7 +11,7 @@ make --trace \
     SAGE_BUILD_PACKAGES='"cd $(SAGE_ROOT) && cd pkg && (../bin/BuildPackages.sh --parallel || true)"' \
     SAGE_STARTED='"$(SAGE_ROOT_BUILD)/gap"' \
     SAGE_REBUILD_CMD='"true"' \
-    SAGE_BUILD_DOC_CMD='"cd $(SAGE_ROOT) && make doc"' \
+    SAGE_BUILD_DOC_CMD?='"true"' \
```

Comment:
@ChrisJefferson I guess you probably still want to make the default behaviour to be to build the manuals? I actually only intended to include the `?=` change in this PR, not to also remove the `make doc`.

In my GAP PR https://github.com/gap-system/gap/pull/4389 I overriding the value of `SAGE_BUILD_DOC_CMD` with `"true"`, so it is irrelevant to me what the default value is.

I'll remove the extra change.